rules/sql.go

```diff
@@ -95,7 +95,7 @@ func (s *sqlStrConcat) checkQuery(call *ast.CallExpr, ctx *gosec.Context) (*gose
 				if op, ok := op.(*ast.Ident); ok && s.checkObject(op, ctx) {
 					continue
 				}
-				return gosec.NewIssue(ctx, be, s.ID(), s.What, s.Severity, s.Confidence), nil
+				return gosec.NewIssue(ctx, query, s.ID(), s.What, s.Severity, s.Confidence), nil
```

Comment:
I am not sure if this is right. I will point to ` db.QueryContext` but the issue might be in the string.